cmd/gardener-controller-manager/app/app.go

```diff
@@ -216,6 +217,41 @@ func run(ctx context.Context, log logr.Logger, cfg *controllermanagerconfigv1alp
 			}
 		}
 
+		managedSeeds := &seedmanagementv1alpha1.ManagedSeedList{}
+		if err := mgr.GetClient().List(ctx, managedSeeds); err != nil {
+			return fmt.Errorf("failed listing objects: %w", err)
+		}
+		if err := meta.EachListItem(managedSeeds, func(o runtime.Object) error {
+			fns = append(fns, func(ctx context.Context) error {
+				obj := o.(client.Object)
+
+				logger := mgr.GetLogger().WithValues("objectKey", client.ObjectKeyFromObject(obj))
+				logger.Info("Check the seed name label for itself")
+				label := v1beta1constants.LabelPrefixSeedName + obj.GetName()
+				logger = logger.WithValues("label", label)
```

Comment:
Do we need this logging? Is this for development? The operation seems simple enough - if the label exists - remove it/send an empty patch. Otherwise, do nothing.

Reply:
https://github.com/gardener/gardener/pull/12029/commits/5216aab4889c1f772b8e9ea4413f93329ebdcc23